src/runtime/HalideRuntime.h

```diff
@@ -1316,8 +1316,11 @@ typedef enum halide_target_feature_t {
     halide_target_feature_sve2,                   ///< Enable ARM Scalable Vector Extensions v2
     halide_target_feature_egl,                    ///< Force use of EGL support.
 
-    halide_target_feature_arm_dot_prod,  ///< Enable ARMv8.2-a dotprod extension (i.e. udot and sdot instructions)
-    halide_target_feature_end            ///< A sentinel. Every target is considered to have this feature, and setting this feature does nothing.
+    halide_target_feature_arm_dot_prod,       ///< Enable ARMv8.2-a dotprod extension (i.e. udot and sdot instructions)
```

Comment:
nit: since the features list is in an arbitrary order, probably cleaner to group all the cuda features together

Reply:
Is it in arbitrary order? I wasn't sure if the target enum values was part of our interface

Reply:
Yes, we absolutely do not promise stability of this enum over multiple releases of Halide